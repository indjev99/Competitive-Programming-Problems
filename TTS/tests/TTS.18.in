34 7
2146.13 -10000
41.57 -10000
10000 -3626.91
2868.92 -10000
-10000 -7804.56
-10000 -9839.53
10000 -6315.17
-6972.9 -10000
1052.69 10000
-10000 -3258.11
10000 2363.36
-3609.61 -10000
10000 -6893.59
-10000 -554.54
-1763.8 10000
10000 3922.45
3247.39 -10000
8796.45 10000
3724.61 -10000
10000 4346.93
10000 2342.08
2537.04 10000
-6255.51 -10000
7845.37 -10000
10000 -519.98
10000 6943.27
1432.89 -10000
-10000 -5607.15
10000 762.36
-10000 1503.34
-4129.9 10000
10000 9166.86
10000 -1803.9
-9365.21 -10000
-2479.21 9687.8
-5264.35 8502.15
-937.03 -9956
-8939.56 -4481.54
9486.55 3163.14
-6659.43 7460.03
-9604.56 2784.3
