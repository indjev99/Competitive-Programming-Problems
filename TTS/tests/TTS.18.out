1930.090000
